src/entrypoint.js

```diff
@@ -72,6 +73,19 @@ async function processIssues(client, args) {
     log.debug(`last updated ${issue.updated_at}`);
     const isPr = 'pull_request' in issue ? true : false;
 
+    if (isPr && args.issueTypes.indexOf(`pull_requests`) == -1) {
```

Comment:
Can I suggest a different input name, maybe "processIssueTypes"?
This line would be less confusing.

Reply:
addressed this by adding consts with better names.